eng/restore/docs.targets

```diff
@@ -9,7 +9,7 @@
           AfterTargets="Restore">
 
     <ItemGroup>
-      <DocFile Include="$(NuGetPackageRoot)$(MicrosoftPrivateIntellisensePackage)/$(MicrosoftPrivateIntellisenseVersion)/xmldocs/netcoreapp/**/*.xml"/>
+      <DocFile Include="$(NuGetPackageRoot)$(MicrosoftPrivateIntellisensePackage)/$(MicrosoftPrivateIntellisenseVersion)/xmldocs/netcoreapp5.0/**/*.xml"/>
```

Comment:
Can this be changed to use $(XmlDocFileRoot)?

Reply:
Actually this change is interesting. You are assuming that the directory name in the intellisense package changed from netcoreapp to netcoreapp5.0. Is this true?

Reply:
@ViktorHofer is right, I was assuming that we'd want to change this in step with TFM but its probably better to just keep it generic (or even remove the folder).

Reply:
cc: @carlossanlop @ahsonkhan we generate this package manually. So if this is going to change we just need to be aware so that we update the folder in the package. 

Also, the docs team is going to be generating a NuGet package for us in the feature.

I would leave it as `netcoreapp`.

Reply:
> I would leave it as `netcoreapp`.

Yep, agreed.